client/src/plugins/version-info/ReleaseInfo.js

```diff
@@ -44,35 +44,39 @@ export function ReleaseInfo(props) {
-          The Modeler will now support you when automating BPMN processes with Camunda 8 by showing implementation errors,
-          and giving hints on how to fix them.
+          <h4>Conditional properties for element templates</h4>
+          This release adds support for conditional properties in your element templates.<br />
+          You can now create dynamic element templates that make the properties panel respond to user input
+          by showing or hiding fields depending on the input of other fields.<br />
+          You can learn more about how to define conditional properties <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/?utm_source=modeler&utm_medium=referral">in the element templates documentation</a>.
         </li>
         <li>
-          <h4>Support drill down into collapsed subprocesses</h4>
-          In Camunda 7 BPMN diagrams, you can now model collapsed subprocesses by
-          collapsing an expanded subprocess. Use the blue drill down icon to view and model the contents of the subprocess.
+          <h4>Adjusted telemetry</h4>
+          We have carefully improved our telemetry to better understand how you use our applications
+          and to support you with a continuously improved modeling experience.<br />
+          You can decide if you want to opt in into our telemetry service or keep your usage data for yourself, of course.<br />
+          A comprehensive overview of what we track, how we do this and why, can be found in our <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/telemetry/">telemetry documentation</a>.
         </li>
         <li>
-          <h4>New customization features for element templates</h4>
-          We added various new element templates features: <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/?utm_source=modeler&utm_medium=referral#groups">custom grouping</a>, <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/?utm_source=modeler&utm_medium=referral">documentation reference</a>, <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/?utm_source=modeler&utm_medium=referral">target element type</a>, and <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/?utm_source=modeler&utm_medium=referral#icons">custom task icons</a>.<br />
-          Discover all configuration options <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/about-templates/?utm_source=modeler&utm_medium=referral">in the element templates docs</a>.
+          <h4>Usability improvements and bug fixes</h4>
+          The section headers of the properties panel are now sticky to provide reliable context
```

Comment:
Instead of going into the details here link the changelog which contains all details.

Reply:
Keep in mind that we also have the CHANGELOG linked per default

![Screenshot_20220710_193442](https://user-images.githubusercontent.com/42800119/178155699-c58b0b6f-aa65-
4816-8ab8-4db632837677.png)

Reply:
Thanks, removed all bug stuff.